website/views/options/social/group.jade

```diff
@@ -51,7 +51,12 @@ a.pull-right.gem-wallet(ng-if='group.type!="party"', popover-trigger='mouseenter
             img.pull-right(ng-show='group.logo', ng-src='{{group.logo}}')
             markdown(text='group.description')
           hr
-          small.muted Group ID: {{group._id}}
+          p=env.t('groupLeader')
+            a(class='badge badge-info', ng-click='clickMember(group.leader, true)')
+              | {{group.leader}}
+          .formatting-help
```

Comment:
Because I liked the amount of space it adds (much less than a `br`), and the `clear: both` doesn't negatively affect anything as far as I can tell.